### PR TITLE
LPS-153241 Ensure unique FDS IDs in client extension module

### PR DESCRIPTION
**Pull request by markocikos**
### References

- [LPS-153241 Ensure unique FDS IDs in client extension module](https://issues.liferay.com/browse/LPS-153241)

### What is the goal of this PR?

1. To ensure uniqueness of FDS ID, we are adding the portlet key prefix.
2. We are removing unnecessary attributes, that are set by default.
3. We are removing the (optimistic) enabling of multiple selection. This will work only if bulk actions are set. I don't think there is a way to set them in the [same way dropdown actions are set](https://github.com/liferay/liferay-portal/blob/master/modules/apps/client-extension/client-extension-web/src/main/java/com/liferay/client/extension/web/internal/frontend/data/set/provider/ClientExtensionEntryFDSActionProvider.java#L57-L71). In a `<headless-display>` tag, both dropdown actions and bulk are typically [set though attributes](https://github.com/liferay/liferay-portal/blob/master/modules/dxp/apps/search-experiences/search-experiences-web/src/main/resources/META-INF/resources/sxp_blueprint_admin/view_sxp_blueprints.jsp#L28-L30). We should probably unify these APIs, but that's a separate task, or epic.
4. We are updating to the new schema builder.

### What does it look like?

No changes in UI.

### Steps to reproduce

Open Applications > Remote Apps

### CODE REVIEW

**Comment by markocikos** [comment]
ci:stop

**Comment by markocikos** [comment]
ci:test:relevant

**Comment by markocikos** [comment]
ci:test:relevant

**Comment by markocikos** [comment]
ci:stop

**Comment by markocikos** [comment]
ci:test:sf

**Comment by markocikos** [comment]
ci:test:relevant

**Comment by markocikos** [comment]
Force-pushed, no changes, retesting.

**Comment by markocikos** [comment]
ci:test:relevant

**Comment by markocikos** [comment]
@diegonvs @beltranrengifo can you take a quick look at this PR? :pray:

File: modules/apps/client-extension/client-extension-web/src/main/resources/META-INF/resources/admin/view.jsp [hunk 1]
```diff
@@ -24,13 +24,8 @@ ClientExtensionAdminDisplayContext clientExtensionAdminDisplayContext = (ClientE
...
 	creationMenu="<%= clientExtensionAdminDisplayContext.getCreationMenu() %>"
 	dataProviderKey="<%= ClientExtensionAdminFDSNames.CLIENT_EXTENSION_ENTRIES %>"
-	formName="fm"
 	id="<%= ClientExtensionAdminFDSNames.CLIENT_EXTENSION_ENTRIES %>"
 	itemsPerPage="<%= 10 %>"
-	namespace="<%= liferayPortletResponse.getNamespace() %>"
-	pageNumber="<%= 1 %>"
-	portletURL="<%= clientExtensionAdminDisplayContext.getCurrentPortletURL() %>"
 	selectedItemsKey="clientExtensionEntryId"
-	selectionType="multiple"
```
Comment (beltranrengifo) [inline modules/apps/client-extension/client-extension-web/src/main/resources/META-INF/resources/admin/view.jsp:34]: I wonder why was this attr active 🤔 
Maybe an unfortunately copy&paste?

**Review by beltranrengifo — APPROVED** [review]
Everything looks fine and safe to me @markocikos 🎉

File: modules/apps/client-extension/client-extension-web/src/main/resources/META-INF/resources/admin/view.jsp (hunk 1 above)
Comment (markocikos) [reply modules/apps/client-extension/client-extension-web/src/main/resources/META-INF/resources/admin/view.jsp:34]: Yeah, probably :slightly_smiling_face:

**Pull request closed without merging**